test/validators.js

```diff
@@ -3,60 +3,10 @@ import fs from 'fs';
 import { format } from 'util';
 import vm from 'vm';
 import validator from '../src/index';
+import { repeat, test } from './testUtils';
```

Comment:
Perhaps this file could just be called `utils`since it's already in the `test` folder?

Reply:
There is already a utils file in the test folder, but maybe we should rename the test file extensions to .test.js

Reply:
`core-test-functions.js`?

Reply:
I went with test-functions and renamed the other files to .test.js